src/components/upnextdialog/upnextdialog.js

```diff
@@ -54,9 +55,9 @@ function setNextVideoText() {
 
     const timeText = '<span class="upNextDialog-countdownText">' + globalize.translate('HeaderSecondsValue', secondsRemaining) + '</span>';
 
-    const nextVideoText = instance.itemType === 'Episode' ?
-        globalize.translate('HeaderNextEpisodePlayingInValue', timeText) :
-        globalize.translate('HeaderNextVideoPlayingInValue', timeText);
+    const nextVideoText = instance.showStaticNextText ?
+        globalize.translate('HeaderNextItem', instance.itemType) :
```

Comment:
itemType should be localized

Reply:
But the rest of the string isn't? What are you suggesting I should be doing instead

Reply:
When calling the `translate` function the first parameter is the key used to lookup a translation and additional parameters replace the placeholders in the translation string, but they are not treated as translation keys themselves. You will need to do something like:

```js
globalize.translate('Key', globalize.translate(itemType));
```

Reply:
Oh my bad. I thought you were saying itemType is ALREADY localized and that I should be doing something else so I can not localizing an already localized string. I'll update this then

Reply:
Updated

Reply:
Ah yeah my bad. I wasn't very clear in my first comment. 😅